featurebyte/routes/block_modification_checker.py

```diff
@@ -0,0 +1,41 @@
+"""
+Block modification checker
+"""
+from typing import Iterator
+
+from contextlib import contextmanager
+
+
+class BlockModificationChecker:
```

Comment:
`Checker` sounds like it only perform checking related tasks. But this class actually control block modification behaviour (either enable or disable the check).

Reply:
hm happy to rename, maybe to `BlockModificationHandler`? any suggestions? 🤔

Reply:
Handler seems better to me 👍